### PR TITLE
#2749 Select all related elements now also selects internal links on FC

### PR DESCRIPTION
**Pull request by etraisnel2**
and PP

PhysicalLinkCategory edges now have the actual pysical links they represent, as semantic elements
as it was already done for FunctionalExchange in ExchangeCategory

Change-Id: Ia0d71a820a1c0e44a7516e346cbba0bbcfb0c5cd

### CODE REVIEW

**Comment by pdulth** [comment]
@etraisnel2 can review as well, i changed code. A Physical Link Category and a Component Category are unoriented. When the the semantic elements was not including both ways of links.

**Review by etraisnel2 — COMMENTED** [review]
Great, works for me

File: tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/common/SelectToolsTest.java [hunk 1]
```diff
@@ -81,16 +140,19 @@ public void test() throws Exception {
     Session s = getSession(getRequiredTestModel());
     SessionContext sc = new SessionContext(s);
 
-    testSelectRelatedFCElements(sc, PAB_DIAGRAM, FC1, FE1, FCAT1, P1, P2, P3, P4);
-    testSelectRelatedPPElements(sc, PAB_DIAGRAM, PPath1, PL1, PCAT1, PP1, PP2, PP3, PP4);
+    testCategoriesSemanticElements(sc);
+    //
+    // testSelectRelatedFCElements(sc, PAB_DIAGRAM, FC1, FE1, FE1_PORT1, FE1_PORT2, FCAT1_EDGE, FCAT1_PORT1,
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/common/SelectToolsTest.java:145]: I hadn't seen. Is it normal? Can you uncomment or remove it?

Comment (etraisnel2) [reply tests/plugins/org.polarsys.capella.test.diagram.tools.ju/src/org/polarsys/capella/test/diagram/tools/ju/common/SelectToolsTest.java]: Not normal , Fixed it

**Pull request merged**